### ISSUE TITLE
Documentation mention parameters `verbose` for torch.optim.lr_scheduler but it does not exist

### ISSUE THREAD

**Issue by alexantoinefortin** [pytorch/pytorch#44080]
## 📚 Documentation

At https://pytorch.org/docs/stable/optim.html, we see that the function `torch.optim.lr_scheduler.MultiStepLR` has a parameter called `verbose` that `If True, prints a message to stdout for each update. Default: False`.

Using `torch.__version__==1.6.0`, 
this minimum example returns an error:

```
import torch.nn as nn
from torch.optim import Adam
from torch.optim.lr_scheduler import MultiStepLR

N, D_in, H, D_out = 64, 1000, 100, 10

model = nn.Sequential(
    nn.Linear(D_in, H),
    nn.ReLU(),
    nn.Linear(H, D_out),
)

optimizer = Adam(model.parameters(), lr=1e-04)
scheduler = MultiStepLR(optimizer, milestones=[20,30], gamma=0.5, verbose=True)
```

Error:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: __init__() got an unexpected keyword argument 'verbose'
```

From the [source code](https://pytorch.org/docs/stable/_modules/torch/optim/lr_scheduler.html#MultiStepLR) (linked in the documentation), this looks generalized to the other learning rate schedulers as well.

cc @jlin27 @vincentqb

**Comment by mariosasko** [comment]
Verbose param was added very recently to the schedulers (last month). It should be there in the nightly release (link to the [source](https://github.com/pytorch/pytorch/blob/master/torch/optim/lr_scheduler.py)). I guess the docs (for version 1.6.0) are not synced properly with the source, but it should be fixed in next stable release.

**Comment by Bharat123rox** [comment]
Fixed by #41580
